Directory.Build.props

```
@@ -0,0 +1,30 @@
+<Project>
+  <PropertyGroup>
+    <ComVisible>false</ComVisible>
+    <CLSCompliant>false</CLSCompliant>
+    <Platforms>AnyCPU;x64;ARM64;x86</Platforms>
+    <AnalysisLevel>preview-all</AnalysisLevel>
+    <LangVersion>preview</LangVersion>
```

We need a fixed language version. 
`preview` version should not be use at project.

use `10` for .Net 6 (lts)

.Net 7 are not LTS Version. So we should not use it on we project.
```suggestion
    <LangVersion>10</LangVersion>
```

---

The recommendation is to not specify it at all so it is coupled with the chosen framework. However this breaks the classic .NET build. I added it conditionally.